plotly/plotly/plotly.py

```diff
@@ -1454,6 +1453,129 @@ def _send_to_plotly(figure, **plot_options):
     return r
 
 
+def create_animations(fig, kwargs, payload):
+    """
+    Makes a post to GRIDS and PLOTS if frames is in the figure.
+
+    This function bypasses the current '/clientresp' route  of making a POST
```

Comment:
Hmm. I don't know that you need to go into the details of `/clientresp`. This doc will likely just become outdated/confusing if you're *too* specific here. Not a big deal.